ee/vellum_ee/workflows/display/utils/events.py

```diff
@@ -29,7 +29,7 @@ def _should_mark_workflow_dynamic(event: WorkflowExecutionInitiatedEvent) -> boo
 
 
 def event_enricher(
-    event: WorkflowExecutionInitiatedEvent, client: Optional[Vellum] = None
+    event: WorkflowExecutionInitiatedEvent, client: Optional[Vellum] = None, metadata: Optional[dict] = None
```

Comment:
update the param of this method to take in any workflow event, and return that same workflow event